src/instr/instr_smpi.h

```diff
@@ -29,7 +29,6 @@ XBT_PRIVATE void TRACE_smpi_testing_out(int rank);
 XBT_PRIVATE void TRACE_smpi_testing_in(int rank, instr_extra_data extra);
 XBT_PRIVATE void TRACE_smpi_alloc();
 XBT_PRIVATE void TRACE_smpi_release();
-XBT_PRIVATE void TRACE_smpi_ptp_in(int rank, const char* operation, instr_extra_data extra);
```

Comment:
Unfortunately, you cannot merge these events. ptp means Point to Point, and that's different from collective. This is true that this is the same behavior for both of them, but actually, each of them should be changed into a signal, at some point.